README.md

```diff
@@ -66,6 +67,30 @@ Then to install buildah follow the steps in this example:
   buildah --help
 ```
 
+In Ubuntu/Debian, you can use this command:
+
+```
+  apt-get -y install software-properties-common
+  add-apt-repository -y ppa:duggan/bats
+  add-apt-repository -y ppa:alexlarsson/flatpak
+  add-apt-repository -y ppa:gophers/archive
+  apt-add-repository -y ppa:projectatomic/ppa
+  apt-get -y -qq update
```

Comment:
Just tested on a UBUNTU server on my work pc and Buildah was up and running afterwards, but this update line threw up a bit.  @lsm5, any thoughts?  I think overall it LGTM and we should ship now and tweak as necessary later on.

root@ubuntu:~# apt-get -y -qq update
W: The repository 'http://ppa.launchpad.net/duggan/bats/ubuntu zesty Release' does not have a Release file.
E: Failed to fetch http://ppa.launchpad.net/duggan/bats/ubuntu/dists/zesty/main/binary-i386/Packages  404  Not Found
E: Some index files failed to download. They have been ignored, or old ones used instead.